README.rst

```diff
@@ -39,13 +39,13 @@ Initializing
 ~~~~~~~~~~~~
 
 Import the package at the top of any file where you will use it. At the very least you will need the
-``OpenTok`` class. Then initialize an OpenTok instance with your own API Key and API Secret.
+``Client`` class. Then initialize an Client instance with your own API Key and API Secret.
```

Comment:
Grammatical error with the new text. Might be worthwhile to throw this through Grammarly just in case, I doubt any of this has every been properly copy-edited.

```suggestion
``Client`` class. Then initialize a Client instance with your own API Key and API Secret.
```